src/components/ToggleButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  ToggleButtonGroup as ToggleButtonGroupPrimitive,
+  ToggleButtonGroupProps as ToggleButtonGroupPrimitiveProps
+} from 'mdlz-prmtz';
+import { transparentize } from 'polished';
+import { theme } from '../theme';
+
+export { ToggleButton } from 'mdlz-prmtz';
+
+export type ToggleButtonGroupProps<T> = ToggleButtonGroupPrimitiveProps<T>
+
+export const ToggleButtonGroup = <T extends string | string[] | null>(
+  props: ToggleButtonGroupProps<T>
+) => (
+  <ToggleButtonGroupPrimitive
+    {...props}
+    styleConfig={{
+      base: {
+        button: {
+          normal: {
+            fontFamily: theme.fonts.normal,
+            height: '36px',
+            marginRight: theme.space[3],
+            padding: '4px 16px',
+            borderRadius: theme.radii[4],
+            background: 'transparent',
+            color: theme.colors.grays[5],
+            fontWeight: 600,
+            fontSize: '14px',
```

Comment:
should we use the `theme.fontSizes[1]` here?

Reply:
Done 👍